src/pineko/fonll.py

```diff
@@ -260,7 +248,8 @@ def collect_updates(fonll_fns, damp):
 def dump_tcards(tcard, tcard_parent_path, theoryid):
     """Produce the seven FONLL theory cards from the original one.
 
-    The produced theory cards are dumped in `tcard_parent_path` with names from '{theoryid}00.yaml' to '{theoryid}06.yaml'.
+    The produced theory cards are dumped in `tcard_parent_path` with names
+    from '{theoryid}00.yaml' to '{theoryid}06.yaml'.
     """
     updates = collect_updates(tcard["FNS"], tcard["DAMP"])
```

Comment:
```suggestion
    updates = collect_updates(tcard["FNS"])
```
this seems correct as right now there will be always 7 theories